tests/virt_autotest/kubevirt_tests_server.pm

```diff
@@ -0,0 +1,255 @@
+    assert_script_run("kubectl apply -f /usr/share/kube-virt/manifests/release/kubevirt-cr.yaml");
+
+    assert_script_run("losetup -a -l"); # check all loop device to see if they refer to deleted files
+    assert_script_run("losetup -D"); # detach all loop device , the disk-image-provider needs to use it to setup images
+    assert_script_run("rm /tmp/hostImages /mnt/local-storage -r"); # remove existing local disks
+    assert_script_run("kubectl apply -f /usr/share/kube-virt/manifests/testing/rbac-for-testing.yaml");
+    assert_script_run("kubectl apply -f /usr/share/kube-virt/manifests/testing/disks-images-provider.yaml");
+    assert_script_run("kubectl apply -f /usr/share/kube-virt/manifests/testing/local-block-storage.yaml");
+
+    assert_script_run("ls /tmp/hostImages/ -l -R");
+    assert_script_run("ls /mnt/local-storage -R -l");
+    assert_script_run("losetup /dev/loop0");
+    assert_script_run("losetup -l -a");
+}
+
+sub run_virt_tests {
+    # Run virt-tests command for each go test files
+    my $cfg = new Config::Simple(data_url("virt_autotest/virt-tests.conf"));
+    my $section;
+    my $key;
+    my ($go_test, $skip_test, $extra_opt);
+    my ($artifacts, $junit_xml, $test_log);
+
+    foreach my $blocks (@kubevirt_tests) {
+        $section = $cfg->get_block($blocks);
+        foreach $key (keys %{$section}) {
+            # print "$key => ${$section}{$key}\n";
+            if ($key == 'GINKGO_FOCUS') {
+                $go_test = ${$section}{$key};
+            } elsif ($key == 'GINKGO_SKIP') {
+                $skip_test = "-ginkgo.skip=\"${$section}{$key}\"";
+            } elsif ($key == 'EXTRA_OPT') {
+                $extra_opt = ${$section}{$key};
+            }
+        }
+        $artifacts = "/tmp/artifacts/$section";
+        $junit_xml = "/tmp/artifacts/$section\_test.xml";
+        $test_log = "/tmp/artifacts/$section\_test.log";
+        script_run("virt-tests -ginkgo.focus=\"$go_test\" \
+                    $skip_test \
+                    -ginkgo.slowSpecThreshold 60 \
+                    -kubeconfig=/root/.kube/config \
+                    -kubectl-path=`which kubectl` \
+                    -virtctl-path=`which virtctl` \
+                    -installed-namespace=kubevirt \
+                    -deploy-testing-infra=false \
+                    -config=/usr/share/kube-virt/manifests/testing/default-config.json \
+                    -ginkgo.v=true \
+                    -test.v=true \
+                    -ginkgo.trace=true \
+                    -ginkgo.noisySkippings=false \
+                    -ginkgo.progress=true \
+                    -ginkgo.noColor \
+                    $extra_opt \
+                    -artifacts=$artifacts \
+                    -junit-output=$junit_xml \
+                    -test.testlogfile=$test_log");
+		
+        assert_script_run("sed -ie 's/Tests Suite/$section/g' $junit_xml");
+    }
```

Comment:
Is there continuous output along with the test run ?

Reply:
Take a screenshot after test run is done

Reply:
It seems that your
 ```
script_run("virt-tests -ginkgo.focus=\"$go_test\" \xxxxxxxxxxxx
```
is fixed also. 
Any need to make it more flexible ? I mean you can the command line can be customized by settings.